tests/lax_test.py

```diff
@@ -1358,6 +1358,18 @@ def fun(operand):
     args_maker = lambda: [rng(shape, dtype)]
     self._CompileAndCheck(fun, args_maker)
 
+  def testReduceWindowFailures(self):
+    def empty_window_test():
+      return lax.reduce_window(np.ones((1,)), 0., lax.add, padding='VALID',
+                               window_dimensions=(0,), window_strides=(1,))
+
+    def zero_stride_test():
+      return lax.reduce_window(np.ones((1,)), 0., lax.add, padding='VALID',
+                               window_dimensions=(1,), window_strides=(0,))
+
+    for failure_fun in [empty_window_test, zero_stride_test]:
+      self.assertRaisesRegex(TypeError, "must have every element be")
```

Comment:
It seems that you forgot the actual function call for `failure_fun`

Reply:
Oops… Fixed it in `2f69ed9`.